docs/standard/cross-platform/index.md

```diff
@@ -25,7 +25,7 @@ You can develop apps for both Microsoft and non-Microsoft platforms by using the
   
 |If you want to...|Use...|  
 |-----------------------|------------|  
-|Share source code between Windows Phone 8.1 and Windows 8.1 apps|**Shared projects** (Universal Apps template in Visual Studio 2013, Update 2).<br /><br /> -   Currently no Visual Basic support.<br />-   You can separate platform-specific code by using #`if` statements.<br /><br /> For details, see:<br /><br /> -   [Build apps that target Windows and Windows Phone by using Visual Studio](http://msdn.microsoft.com/library/windows/apps/dn609832.aspx) (MSDN article)<br />-   [Using Visual Studio to build Universal XAML Apps](http://blogs.msdn.com/b/visualstudio/archive/2014/04/14/using-visual-studio-to-build-universal-xaml-apps.aspx) (blog post)<br />-   [Using Visual Studio to Build XAML Converged Apps](http://channel9.msdn.com/Events/Build/2014/3-591) (video)|  
+|Share source code between Windows Phone 8.1 and Windows 8.1 apps|**Shared projects** (Universal Apps template in Visual Studio 2013, Update 2).<br /><br /> -   Currently no Visual Basic support.<br />-   You can separate platform-specific code by using #`if` statements.<br /><br /> For details, see:<br /><br /> -   [Build apps that target Windows and Windows Phone by using Visual Studio](https://msdn.microsoft.com/library/windows/apps/dn609832.aspx) (MSDN article)<br />-   [Using Visual Studio to build Universal XAML Apps](http://blogs.msdn.com/b/visualstudio/archive/2014/04/14/using-visual-studio-to-build-universal-xaml-apps.aspx) (blog post)<br />-   [Using Visual Studio to Build XAML Converged Apps](https://channel9.msdn.com/Events/Build/2014/3-591) (video)|  
```

Comment:
can you update this http link to be https://blogs.msdn.microsoft.com/visualstudio/2014/04/14/using-visual-studio-to-build-universal-xaml-apps/?